style.css

```diff
@@ -0,0 +1,41 @@
+.img-wrapper{
+    margin: var(--spacing6) 0;
+    display: flex;
+    justify-content: center;
+}
+
+img {
+    width: 35rem;
+    height: 30rem;
+}
```

Comment:
this will effect all images globally, so add a class

Reply:
done the changes as suggested by you. Please check now